package.json

```diff
@@ -9,7 +9,18 @@
     "test": "lerna run test --stream --concurrency 1",
     "lint": "eslint **/*.js -c .eslintrc.json && markdownlint **/*.md --ignore **/node_modules/**",
     "screenshots": "lerna run screenshots --stream --concurrency 1",
-    "start": "lerna run bundle && npm start --prefix ./packages/d3fc"
+    "start": "lerna run bundle && npm start --prefix ./packages/d3fc",
+    "commitlint": "commitlint --from HEAD~1 to HEAD --verbose"
+  },
+  "husky": {
+    "hooks": {
+      "prepare-commit-msg": [
+        "echo +----------------------------------------------+ &&",
+        "echo.^|      This repo is Commitzen Friendly!        ^| &&",
+        "echo.^| Use git cz to generate a conventional commit ^| &&",
```

Comment:
I think `git cz` assumes that they have installed commitizen globally? 

Is it worth recommending `npx git-cz` instead?

Reply:
That's right, nice catch, thanks